phpBB/phpbb/db/migration/data/v330/jquery_update3.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+*
+* This file is part of the phpBB Forum Software package.
+*
+* @copyright (c) phpBB Limited <https://www.phpbb.com>
+* @license GNU General Public License, version 2 (GPL-2.0)
+*
+* For full copyright and license information, please see
+* the docs/CREDITS.txt file.
+*
+*/
+
+namespace phpbb\db\migration\data\v330;
+
+class jquery_update3 extends \phpbb\db\migration\migration
+{
+	public function effectively_installed()
+	{
+		return $this->config['load_jquery_url'] !== '//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js';
```

Comment:
Is the desire with this that if someone had decided to update to a newer restaurant different version at some point that this migration won’t run? Because that would be the result.

Reply:
This should be for version 1.12.4

Reply:
if this is true then its been a problem forever 🤣

Reply:
Not sure what you mean, but as it stands, this migration will not update sh@t unless you change this value to exactly match the value this migration is updating to, so 1.12.4.

Reply:
```
class jquery_update2 extends \phpbb\db\migration\migration
{
	public function effectively_installed()
	{
		return $this->config['load_jquery_url'] !== '//ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js';
	}

	static public function depends_on()
	{
		return array(
			'\phpbb\db\migration\data\v310\jquery_update',
		);
	}

	public function update_data()
	{
		return array(
			array('config.update', array('load_jquery_url', '//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js')),
		);
	}

}
```

Reply:
Yeah I think it should be this, going forward:
```
return $this->config['load_jquery_url'] === '//ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js';
```
It's the same logic, but this way the migration author only needs to be concerned with the current version, and not hunt for the exact previous version.

Notice I changed the logic from `!==` to `===` and of course the version number

Reply:
ok